.github/workflows/release.yml

```diff
@@ -57,12 +58,22 @@ jobs:
             echo "dry-run=false" >> $GITHUB_OUTPUT
             echo "Setting dry-run to FALSE"
           fi
+      - name: Evaluate Multi-Arch
+        id: multi-arch
+        run: |
+          if [ "${{ inputs.multi-arch }}" = "true" ]; then
```

Comment:
uberest nit but can we consolidate on `[[ foo == bar ]]` instead of `[ foo = bar ]`? the latter works but it's confusing to folks

Reply:
Yes, and then I'll run another dry-run. Thanks!